batcher/aligned-sdk/src/sdk.rs

```diff
@@ -825,50 +804,3 @@ fn save_response_json(
 
     Ok(())
 }
-
-#[cfg(test)]
```

Comment:
Shouldn't we rewrite the tests for `max_fee_per_proof` instead of removing them?

Reply:
I am in favor of just removing them as they rely on an rpc and could be flaky when used on the ci.

Reply:
If that's the case, I would suggest a different approach. Just extract the computation to a helper and test the helper. Then the RPC call will be untested by unit tests but it will necessarily be tested on integration ones (as the function is used). The latter don't guarantee we use the right numbers for the right case.

Reply:
It's a bit like mocking but without lying to ourselves about having tested the interaction with the RPC.